src/vireshJJun20/Assignment19/ArrayComparison3.java

```diff
@@ -0,0 +1,28 @@
+package vireshJJun20.Assignment19;
+
+public class ArrayComparison3 { //// returns the maximum value from mismatching index post comparing 2 arrays.
+	int index= 0;
+	void arrayIndex(int[] ar1, int[] ar2) {
+		if (ar1.length==ar2.length) {
+			for (int i= 0; i<ar2.length; i++) {
+				if(ar1[i]!=ar2[i]) {
+					int max=0;
```

Comment:
you should define max = ar1[2] initially, else is not required.

Reply:
Ok Krishna, will take care from next time. Thanks.